beetsplug/play.py

```diff
@@ -111,22 +115,29 @@ def play_music(self, lib, opts, args):
             if ui.input_options(('Continue', 'Abort')) == 'a':
                 return
 
-        # Create temporary m3u file to hold our playlist.
-        m3u = NamedTemporaryFile('w', suffix='.m3u', delete=False)
-        for item in paths:
-            if relative_to:
-                m3u.write(relpath(item, relative_to) + b'\n')
-            else:
-                m3u.write(item + b'\n')
-        m3u.close()
-
         ui.print_(u'Playing {0} {1}.'.format(len(selection), item_type))
+        if raw:
+            passed_to_command = paths
+        else:
+            passed_to_command = self._create_tmp_playlist(paths)
```

Comment:
Here, `args` (or `open_args` or something) does seem like the right variable name.